docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -313,23 +313,49 @@ go get github.com/hazelcast/hazelcast-go-client
 +
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
+
+import (
+  "context"
+  "fmt"
+  "log"
 
-func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+  "github.com/hazelcast/hazelcast-go-client"
+)
 
-  cb.Cluster().SetName("hello-world") <1>
+func main() {
+  ctx := context.TODO()
 
-  client, err := hazelcast.StartNewClientWithConfig(cb) <2>
+  cb := hazelcast.NewConfig()
+  cb.ClientName = "hello-world" <1>
```

Comment:
Java client sample seems to set the cluster name. Why do we set the client name here? Maybe the default cluster name  can be used: `dev`.

Reply:
Tutorial is running the hz docker image with `-e HZ_CLUSTERNAME=hello-world \` option. Therefore, as other clients did, setting the cluster's name `hello-world` is a better option than dev I guess.